src/scripting.c

```diff
@@ -1206,13 +1203,13 @@ void scriptingInit(int setup) {
     /* Replace math.random and math.randomseed with our implementations. */
     lua_getglobal(lua,"math");
 
-    lua_pushstring(lua,"random");
-    lua_pushcfunction(lua,redis_math_random);
-    lua_settable(lua,-3);
-
-    lua_pushstring(lua,"randomseed");
-    lua_pushcfunction(lua,redis_math_randomseed);
-    lua_settable(lua,-3);
+//    lua_pushstring(lua,"random");
+//    lua_pushcfunction(lua,redis_math_random);
+//    lua_settable(lua,-3);
+//
+//    lua_pushstring(lua,"randomseed");
+//    lua_pushcfunction(lua,redis_math_randomseed);
+//    lua_settable(lua,-3);
```

Comment:
Did you mean to leave these commented out? I think they're causing a test failure.